### ISSUE TITLE
Selected OS is difficult to see for me

### ISSUE THREAD

**Issue by tbrowder** [Raku/rakubrew.org#19]
If possible, I would like to see more differentiation between the selected OS and the other two. Perhaps use underlining as well as a green color instead of red.

Then, maybe add something like this to the right side on the same line:

    <== install on Windows [or Mac or Linux]

My observation is that it is not an easy page to decipher for an inexperienced Windows computer user who may be a beginning student in elementary school.

**Comment by patrickbkr** [comment]
Just to make sure we have the same context: Are you aware that visiting rakubrew.org will auto-detect the right operating system and in the most common case it's not necessary to pay attention to the OS selector?

**Comment by tbrowder** [comment]
Yes, but I think it is difficult to see or understand that, especially for sight issues (I have less-than-desired color vision) or inexperienced computer users.
